### PR TITLE
fix: perm for any repo

### PR DESCRIPTION
**Pull request by olenagerasimova**
part of #501 

Added fix and tests for the case with `adapter_basic_permission` for any repository.

### CODE REVIEW

**Comment by olenagerasimova** [comment]
@dgarus please, check, that's not big but important fix

File: src/test/java/com/artipie/security/policy/CachedYamlPolicyTest.java [hunk 1]
```diff
@@ -347,4 +406,48 @@ private byte[] tester() {
             "        - download"
         ).getBytes(StandardCharsets.UTF_8);
     }
+
```
Comment (dgarus) [inline src/test/java/com/artipie/security/policy/CachedYamlPolicyTest.java:409]: all these methods can be replaced by one
```
private byte[] userConfig(String user, String repo){
        String template = String.join(
            "\n",
            "%s:",
            "  permissions:",
            "    adapter_basic_permission:",
            "      %s:",
            "        - download"
        );
        return String.format(template, user, repo).getBytes(StandardCharsets.UTF_8);
    }
    
```

**Review by dgarus — CHANGES_REQUESTED** [review]
@olenagerasimova thanks!
Please check my comment.

Do we really need a username in the permissions file? 
It looks like an info duplication because we already have username as file name.

**Comment by olenagerasimova** [comment]
@dgarus thanks, corrected

**Review by dgarus — APPROVED** [review]
thanks!

**Pull request merged**